### DISCUSSION TITLE
Discussion on `OpenAPITool`

### DISCUSSION THREAD

**Discussion by TuanaCelik** [deepset-ai/haystack-experimental#79] [General]
This is the discussion board for the experimental `OpenAPITool` component.

The `OpenAPITool` calls a RESTful endpoint of an OpenAPI service using payloads generated from human instructions.

📚 **[Full Documentation](https://docs.haystack.deepset.ai/reference/openapi-api)**

🧑‍🍳 **[Try out the Cookbook](https://colab.research.google.com/github/deepset-ai/haystack-cookbook/blob/main/notebooks/openapitool.ipynb)**

Please try this component out, let us know how you find it, what you find easy or difficult.

> **Note:** Experimental features live in this repository for a fixed period of time. We don't guarantee that we will continue maintaining experimental features. But if they are successful, stable and if you like it, we will move the feature to the core Haystack package.

**Comment by bilgeyucel** [comment]
[`OpenAPITool`](https://github.com/deepset-ai/haystack-experimental/blob/v0.3.0/haystack_experimental/components/tools/openapi/openapi_tool.py) is discontinued and removed as of `haystack-experimental==0.4.0`. As an alternative, you can use [openapi-llm](https://github.com/vblagoje/openapi-llm).
